commit 44ccfc47d7d1b8b69d9e1c63b1fa2bade7d8083f (refs/changes/19/1319/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-01-23T11:44:43-08:00 (2 years, 8 months ago)
    
    PUBAPI-1358 cloudapi's usage of sdc-clients' IMGAPI client is working by luck
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/package.json b/package.json
index c9ec288..720149a 100644
--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
         "bunyan": "1.8.1",
         "cueball": "1.1.9",
         "kang": "1.1.0",
-        "sdc-clients": "10.0.0",
+        "sdc-clients": "10.0.4",
         "ufds": "1.2.0",
         "semver": "2.2.1",
         "nodemailer": "0.3.29",
